src/Microsoft.PowerShell.Commands.Management/commands/management/ComputerUnix.cs

```diff
@@ -105,7 +105,7 @@ public class CommandLineCmdletBase : PSCmdlet, IDisposable
         /// </summary>
```

Comment:
@vexx32, your last commit had 3 failures in `PowerShell-CI-linux`
Invoke-WebRequest tests.Error occurred in Describe block
```powershell
WebListener did not start before the timeout was reached.
Errors:

&27;[91mGet-Command: &27;[91mThe term 'WebListener' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.&27;[0m

&27;[91mSplit-Path: &27;[91mCannot bind argument to parameter 'Path' because it is null.&27;[0m

&27;[91mInvalidOperation: &27;[91mThe expression after '&' in a pipeline element produced an object that was not valid. It must result in a command name, a script block, or a CommandInfo object.&27;[0m

Output:
appEXE: 
serverPfxPath: /tmp/ServerCert.pfx
serverPfxPassword: 3aebc4ca7b
HttpPort: 8083
Https: 8084
Tls11Port: 8085
TlsPort: 8086

Verbose:

At /home/vsts/work/1/s/test/tools/Modules/WebListener/WebListener.psm1:177 char:13
```
Invoke-RestMethod tests.Error occurred in Describe block
```powershell
WebListener did not start before the timeout was reached.
Errors:

&27;[91mGet-Command: &27;[91mThe term 'WebListener' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.&27;[0m

&27;[91mSplit-Path: &27;[91mCannot bind argument to parameter 'Path' because it is null.&27;[0m

&27;[91mInvalidOperation: &27;[91mThe expression after '&' in a pipeline element produced an object that was not valid. It must result in a command name, a script block, or a CommandInfo object.&27;[0m

Output:
appEXE: 
serverPfxPath: /tmp/ServerCert.pfx
serverPfxPassword: 2a6346a19e
HttpPort: 8083
Https: 8084
Tls11Port: 8085
TlsPort: 8086

Verbose:

At /home/vsts/work/1/s/test/tools/Modules/WebListener/WebListener.psm1:177 char:13
```
Validate Invoke-WebRequest and Invoke-RestMethod -InFile.Error occurred in Describe block
```powershell
WebListener did not start before the timeout was reached.
Errors:

&27;[91mGet-Command: &27;[91mThe term 'WebListener' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.&27;[0m

&27;[91mSplit-Path: &27;[91mCannot bind argument to parameter 'Path' because it is null.&27;[0m

&27;[91mInvalidOperation: &27;[91mThe expression after '&' in a pipeline element produced an object that was not valid. It must result in a command name, a script block, or a CommandInfo object.&27;[0m

Output:
appEXE: 
serverPfxPath: /tmp/ServerCert.pfx
serverPfxPassword: 5aae193e20
HttpPort: 8083
Https: 8084
Tls11Port: 8085
TlsPort: 8086

Verbose:

At /home/vsts/work/1/s/test/tools/Modules/WebListener/WebListener.psm1:177 char:13
```